cmd/livepeer/livepeer.go

```diff
@@ -164,6 +164,7 @@ func parseLivepeerConfig() starter.LivepeerConfig {
 
 	// Live AI Media Server:
 	cfg.MediaMTXApiPassword = flag.String("mediaMTXApiPassword", "", "HTTP basic auth password for MediaMTX API requests")
+	cfg.LiveAITrickleHostForRunner = flag.String("liveAITrickleHostForRunner", "", "Trickle Host used by AI Runner; It's used to overwrite the publicly available Trickle Host")
```

Comment:
If docker is a standard configuration maybe we should place `172.17.0.1:8888` as a default in here?

Reply:
I think there are many different setups, so I think the default should be what you expose as a public IP. I believe that if we make `172.17.0.1:8888` default, then it becomes more confusing.